src/main/java/uk/nhs/digital/nhsconnect/nhais/model/fhir/AmendmentPatchOperation.java

```diff
@@ -0,0 +1,18 @@
+package uk.nhs.digital.nhsconnect.nhais.model.fhir;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@RequiredArgsConstructor
+@ToString
+@Getter
+public enum AmendmentPatchOperation {
+    ADD("add"),
+    REPLACE("replace"),
+    REMOVE("remove");
```

Comment:
in specification there is also operation "test"

Reply:
We don't allow it. Have a look at the OpenAPI for amendment. It describes the additional restrictions we put on amendments.

Reply:
ok, thank you @mattd-kainos, I've been looking on https://portal.developer.nhs.uk/ only